qemu/tests/qemu_guest_agent.py

```diff
@@ -1829,25 +1832,128 @@ def gagent_check_nonexistent_cmd(self, test, params, env):
             test.fail("Should return error info.")
 
     @error_context.context_aware
-    def gagent_check_with_migrate(self, test, params, env):
```

Comment:
@xiagao Why delete this function?

Reply:
@vivianQizhu  I'm supposing it's branch conflict. And now it's back.